py-polars/polars/dataframe/frame.py

```diff
@@ -840,7 +840,7 @@ def _read_parquet(
             else:
                 raise ValueError(
                     "cannot use glob patterns and integer based projection as `columns`"
-                    " argument; Use columns: List[str]"
+                    "\n\nargument; Use columns: List[str]"
```

Comment:
I think you put the newlines in the wrong place, should it not go after "argument"? Same with 952 below.

Reply:
Edit: I see this was mentioned ago, and you regex'd this which makes sense. Carry on!